apps/server/src/modules/tool/external-tool/uc/external-tool.uc.spec.ts

```diff
@@ -468,4 +481,77 @@ describe('ExternalToolUc', () => {
 			expect(externalToolService.deleteExternalTool).toHaveBeenCalledWith(toolId);
 		});
 	});
+
+	describe('getMetadataForExternalTool', () => {
```

Comment:
please rewrite the test for this method in a way that is compliant to the style guide